File: csharp/duplicate.cs
//{ Driver Code Starts
//Initial Template for C#

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DriverCode
{

    class GFG
    {
        static void Main(string[] args)
        {
            int testcases;// Taking testcase as input
            testcases = Convert.ToInt32(Console.ReadLine());
            while (testcases-- > 0)// Looping through all testcases
            {
                int N = Convert.ToInt32(Console.ReadLine());// Input for size of array
                int[] arr = new int[N];

                string[] stringArray = Console.ReadLine().Split(' ');
                for (int i = 0; i < N; i++)
                {
                    arr[i] = int.Parse(stringArray[i]);
                }

                Solution obj = new Solution();
                int[] res = obj.duplicates(arr, N);
                for(int i = 0;i<res.Length;i++){
                    Console.Write(res[i]+" ");
                }
                Console.Write("\n");

            }

        }
    }

    
// } Driver Code Ends
//User function Template for C#

class Solution
    {
        //Complete this function
        //Function to check if there is a pair with the given sum in the array.
        public int[] duplicates(int[] arr, int n)
        {
            //Your code here
            int aux[];
            foreach (var a in arr)
            {
               int current = arr[i];
               aux = arr.Select(x => x == current).ToList()
            }
            
        }

    }

//{ Driver Code Starts.
    
}
// } Driver Code Ends

File: c/duplicate.c
#include <stdio.h>


void main(void)
{

    int N = 10;
    int arr[] = {1,2,3,4,44,39,74,8,7,9};
    int aux[sizeof(arr)];
    int aux_index = 0;
   
    for(int i = 0 ; i < N; i++)
    {
        int current = arr[i];
        for(int j = i+1 ; j < N; j++)
        {
            if(arr[j] == current)
            {
                aux[aux_index] = current;
                aux_index++;
            }
        }
    }
        
        if(aux_index != 0)
        {
         for(int i = 0 ; i < N; i++)
             printf("%d , ",aux[i]);
        }
        else printf("%d",-1);

    return;    

}